package.json

```diff
@@ -2,10 +2,10 @@
   "name": "kiteconnect",
   "version": "4.1.0",
   "description": "The official JS client library for the Kite Connect trading APIs",
-  "main": "./lib/",
+  "main": "./dist/",
   "scripts": {
     "generate-docs": "node_modules/.bin/jsdoc --configure .jsdoc.json --verbose",
```

Comment:
`generate-docs` script does not work anymore. Probably the target folder in `.jsdoc.json` needs to be updated.

Also, there is no complication script, which should be there maybe?